packages/cli/src/commands/dev.js

```diff
@@ -20,9 +20,11 @@ export const handler = async ({ app = ['db', 'api', 'web'] }) => {
   const API_DIR = path.join(BASE_DIR, 'api')
   const WEB_DIR = path.join(BASE_DIR, 'web')
 
-  // Generate the Prisma client if it doesn't exist.
-  await generatePrismaClient({ verbose: false, force: false })
-
+  // Generate the Prisma client if it doesn't exist
+  // but only in case of db command option and API dir present
+  if (app.includes('db') && fs.existsSync(API_DIR)) {
+    await generatePrismaClient({ verbose: false, force: false })
+  }
```

Comment:
This was originally added because the `api` dev server couldn't launch unless the database client was generated. It _might_ be the case that that's no longer true, and then we can remove this.

The way to test would be to ensure that `node_modules/@prisma/client` isn't generated, or if it's the latest version then delete `node_modules/.prisma`